content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -208,6 +208,15 @@ The following manifest defines an Ingress that sends traffic to your Service via
    If you are running Minikube locally, use `minikube ip` to get the external IP.
    The IP address displayed within the ingress list will be the internal IP.
    {{< /note >}}
+   
+   {{< note >}}
+   For Mac, if you are running Minikube locally with Docker, point to localhost in the hosts file instead of the minikube ip
```

Comment:
```suggestion
   For Mac, if you are running Minikube locally with Docker Desktop, point to the loopback IP address
   in the hosts file instead of the minikube IP address. For example:
```